### PR TITLE
KEYCLOAK-15592: Fix removing KEYCLOAK_ADAPTER_STATE cookie

### PR DESCRIPTION
**Pull request by michalm86**
This commit provides a fix for removing KEYCLOAK_ADAPTER_STATE cookie
by using 'path' and 'domain' parameters properly.

The issue is reported here: https://issues.redhat.com/browse/KEYCLOAK-15592

### CODE REVIEW

**Review by mhajas — CHANGES_REQUESTED** [review]
Thank you very much for the PR @michalm86! The change looks good to me, I compared it to wildfly adapter and we are doing there exactly what you suggested. However, we can't merge this unless there are some tests. 

I looked at our tests and there are some tests around this cookie [here](https://github.com/keycloak/keycloak/blob/master/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/servlet/DemoServletsAdapterTest.java#L1370).

Also it seems all of them are [ignored](https://github.com/keycloak/keycloak/blob/master/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/servlet/jetty/JettyDemoServletAdapterTest.java) on Jetty and I am wondering whether the reason for ignoring isn't the issue you just fixed. Would you mind to give it a try and check these tests on Jetty? You can find more details on how to run testsuite [here](https://github.com/keycloak/keycloak/blob/master/testsuite/integration-arquillian/HOW-TO-RUN.md#jetty).

**Comment by stianst** [comment]
Closing due to lack of tests, and no feedback. Feel free to re-open if you can provide some tests.

**Pull request closed without merging**
